CHANGELOG.md

```diff
@@ -49,12 +49,14 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
 > 
 > If your organization doesn't currently have an Enterprise plan, you can test out this functionality by signing up for a free Enterprise trial.
 
-> The `@policy` directive requires using a federation version not yet available at the time of router release `1.34.0`.
+> The `@policy` directive requires using a [federation version `2.6`](https://www.apollographql.com/docs/federation/federation-versions).
```

Comment:
When this goes live, it needs to be updated in [releases](https://github.com/apollographql/router/releases) as well.

Reply:
That'll be updated with the new router release